### ISSUE TITLE
Color picker blocks the render view

### ISSUE THREAD

**Issue by strattonbrazil** [OpenShot/openshot-qt#3799]
**Describe the bug:**
The dialog picker pops up in the middle of the app window and moves the window with it, it's always blocking the render view. That makes it impossible to color select anything in that region (see example screenshot with purple outline). 

**Steps to reproduce the behavior click behind behavior:**
1. Loading OpenShot
2. Load color picker from effect property (ex. Chrome Key color)
3. Try moving the color picker away from the render view

**Expected behavior:**
Color picker can be moved out of the way and any portion of the render view can be selected

**System Details:**
- OpenShot Version: 2.5.1
- Operating System / Distro: Ubuntu 18.04

**Screenshot:**
![picking_behind_dialog](https://user-images.githubusercontent.com/786471/97397779-491a5680-18a7-11eb-98a8-e6a1fa9a8a4e.png)

**Comment by ferdnyc** [comment]
@strattonbrazil  Yeah, that was a tricky one.

Because the Animated Titles interface itself is _already_ a dialog window, Qt really didn't want to go popping up _another_ dialog window completely independent of its parent dialog window.

But I believe I managed to convince it, in PR #3805. Works for me testing on Fedora 32.

**Comment by ferdnyc** [comment]
Hah! OK, well, maybe not so fast.

When I read "render view", I thought you were referring to the Blender animated-titles renderer, so that's where I fixed this. But you were talking about the preview in the main window, when choosing colors for effects. That's totally my fault for not reading your report carefully enough, and 
I apologize.

The issue you were reporting is not _yet_ fixed, as of the merged PR. But I've already concluded it makes sense to fix it there as well (and in the non-animated Title Editor), the other two places we use color-picker dialogs. And the good news is, I already have the code written, so it's halfway there.

I just need to break out the code I used to fix this in the Blender dialog, and also use it in the other two interfaces as well. Shouldn't take too long to have it working everywhere.
